src/framework/accordion/Accordion.spec.jsx

```diff
@@ -0,0 +1,53 @@
+
+import { TestCaseFactory } from 'react-test-kit';
+import Accordion from './Accordion.jsx';
+
+describe('Accordion', () => {
+  describe('Props', () => {
```

Comment:
Could start testing the structure of code as well (like we do in SEM). Let's talk about this point.